doc/installer-hacks.md

```diff
@@ -105,3 +105,17 @@ equivalent, the old code is simply deleted, not commented.
 
 * Code dealing with BIOS-ID changed to blindly assume common values. Original
   code commented and marked with "storage-ng".
+
+## Changes in autoyast2-installation
```

Comment:
I would say autoyast2. That's how the package is called at OBS. autoyast2-installation is just a subpackage and the changes affect the whole OBS package.